### ISSUE TITLE
Onnx softplus accuracy problem

### ISSUE THREAD

**Issue by Shawn314** [microsoft/onnxruntime#10491]
**Describe the bug**
I found a accuracy problem about onnx softplus. Onnx softplus operator behavior is inconsistent with other framework calculation results.

**Urgency**
None.

**System information**
- OS Platform and Distribution : Linux Ubuntu 18.04
- ONNX Runtime installed from (source or binary): binary
- ONNX Runtime version: 1.10.2
- Python version: 3.8.10

**To Reproduce**
- A scalar is **-12.3392**
- pytorch and tensorflow softplus result is 4.37676e-6, but onnx is 4.41073e-6.

**Expected behavior**
All Inference framework has same result.

**Screenshots**
Pytorch
![image](https://user-images.githubusercontent.com/30689370/152963056-44896553-c3f4-4adb-a16c-c91db668345a.png)
Tensorflow
![image](https://user-images.githubusercontent.com/30689370/152963165-755510e4-1504-48ec-98dc-5d8ba01a256c.png)
Onnx
![image](https://user-images.githubusercontent.com/30689370/152963359-4b8644f4-d904-4ced-970d-c4f7750bd56b.png)

**Additional context**
None
